plastic_detection_service/reproject_raster.py

```diff
@@ -47,24 +49,27 @@ def raster_to_wgs84(input_raster: bytes) -> gdal.Dataset:
     osr.CoordinateTransformation(srs_utm, srs_wgs84)
 
     out_path_memory = "/vsimem/temp.tif"
+
     out_ds: gdal.Dataset = gdal.Warp(
```

Comment:
When executing I get the following Error in line 53: TypeError: WarpOptions() got an unexpected keyword argument 'srcBands'. It's probably a version mismatch- we should write down in our readme which gdal versions we use? I'm on version 3.4.3 and you?:)

Reply:
im on GDAL 3.7.1 and python gdal=3.7.1.1